src/tests/ftest/util/command_utils.py

```diff
@@ -210,7 +210,13 @@ def stop(self):
                     self._command, str(state))
                 self._process.send_signal(signal_to_send)
                 if signal_list:
-                    time.sleep(5)
+                    start = time.time()
+                    while time.time() - start < 5:
+                        time.sleep(0.01)
+                        if self._process._popen.poll():
+                            break
```

Comment:
We could save an extra 0.01 seconds:
```suggestion
                    while self._process._popen.poll() is None and time.time() - start < 5:
                        time.sleep(0.01)
```

Reply:
Agreed.  Locally I was seeing it typically take about 1s, down from 5 so I think that's unlikely to help further but it's certainly cleaner.